package.json

```diff
@@ -17,9 +17,7 @@
     "react-native-onesignal": "^1.2.3"
   },
   "devDependencies": {
-    "rollup": "^0.42.0",
-    "rollup-plugin-commonjs": "^8.0.2",
-    "rollup-plugin-fable": "^1.0.6",
-    "rollup-watch": "^4.0.0"
+    "babel-core": "^6.25.0",
```

Comment:
As commented above, for RN it's not necessary to compile ES2015 to ES5, so no config file is needed (well the Babel config is actually the `babel` section of splitter.config.js) but Babel is still necessary, because Fable only outputs a Babel-compatible AST and Babel must convert it into actual JS code.